frontend/src/components/Header/components/CuratorAPIKeyGenerator/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Dialog, Intent, MenuItem } from "@blueprintjs/core";
```

Comment:
`czif` also exports `Dialog` that you can use!

Reply:
Dialog wasn't play well with the bp3 menu, a click on the Dialog would close the menu and would then close the dialog

Reply:
When we're dealing with >1 design system on a file should we import all SDS comports as `SDS*`

Reply:
Hmm I think unless there's name collision, we probably don't need the namespace?

Reply:
Hmm let me take a look at the Dialog issue now. Thanks for letting me know!

Reply:
Somehow it works for me!

![demo](https://user-images.githubusercontent.com/6309723/156634943-00dfd0e1-6e4e-4744-b1ae-593371d9b05c.gif)


<img width="849" alt="Screen Shot 2022-03-03 at 11 06 31 AM" src="https://user-images.githubusercontent.com/6309723/156634879-71d002d9-0681-43e1-ba96-82b2912136a6.png">

Reply:
I think you just happen to be clicking on things that are supposed to close the dialog

Uploading Screen Recording 2022-03-03 at 1.30.31 PM.mov…

Reply:
Oh!! I see what you mean now. Interesting 🤔 Yeah let's use Blueprint's Dialog for now and maybe a followup PR later if we find a solution to this!